### PR TITLE
ICU-21078 Improve instructions and gitignore file for cldr-to-icu

### PR DESCRIPTION
**Pull request by hugovdm**
##### Checklist

- [X] Issue filed: https://unicode-org.atlassian.net/browse/ICU-21078
- [X] Updated PR title and link in previous line to include Issue number
- [X] Issue accepted
- [ ] Tests included
- [X] Documentation is changed or added

### CODE REVIEW

File: tools/cldr/.gitignore [hunk 1]
```diff
@@ -3,5 +3,5 @@
 !/lib/README.txt
 
 # Ignore the default Maven target directory.
-/target
```
Comment (hagbard) [inline tools/cldr/.gitignore:6]: Good pont. Can you also add any occurrence of directory /.idea/ here?
It's what IntelliJ uses.

Comment (hugovdm) [reply tools/cldr/.gitignore:6]: Where do these directories appear? If I make it "/.idea", I think it means only ".idea" in the same directory as the gitignore file. If I make it ".idea", it's also in the subdirectories.

Comment (hugovdm) [reply tools/cldr/.gitignore:6]: Added `.idea/` in the top-level .gitignore file, next to `.vscode/`.

**Comment by hugovdm** [comment]
I squashed commits, this being standard procedure in CLDR. It looks like this looses me my approval though?
"@hagbard approved this pull request."

**Pull request merged**